larq_compute_engine/mlir/ir/lce_ops.td

```diff
@@ -57,8 +60,9 @@ TODO
   let arguments = (ins
     TensorOf<[F32]>:$input,
     TensorOf<[F32, I32]>:$filter,
-    TensorOf<[F32]>:$post_activation_multiplier,
-    TensorOf<[F32]>:$post_activation_bias,
+    TensorOfOrNone<[F32]>:$post_activation_multiplier,
+    TensorOfOrNone<[F32]>:$post_activation_bias,
+    TensorOfOrNone<[I32]>:$write_bitpacked_threshold,
```

Comment:
I am not sure if I like the name `write_bitpacked_threshold`, it seems to suggest that it is a boolean that decides wether to write a bitpacked threshold or not. How do you feel just calling it `threshold`?
@koenhelwegen you are usually good in coming up with names for those things. Do you think `threshold` is clear enough?

This is not a blocker for this PR, since it will be easier to change once it is merged into #387.

Reply:
Yeah I agree the name is quite clunky, definitely open to naming suggestions.

Reply:
`threshold` sounds reasonable to me, or `quantization_threshold`/`binarization_threshold`/`output_threshold`? But I'm on unfamiliar territory here so not sure what works in this context. Perhaps `bitpacking_threshold` (instead of `bitpacked`) would avoid the confusion @lgeiger raised?

Reply:
I think I'd prefer `output_threshold` as well. I only just now realized that the naming decision isn't really that important here since the kernel will just use the 4th operand and doesn't rely on the name at all since it is not an attribute.

> But I'm on unfamiliar territory here so not sure what works in this context.

That's the reason why I asked for your opinion, since I thought this is an attribute name that would be exposed to users :)